### ISSUE TITLE
Slidable header - horizontal menu navbar 

### ISSUE THREAD

**Issue by breakinho** [start-angular/SB-Admin-BS4-Angular-5#200]
I am working on that template, and i want to get rid off sidebar, and made it horizontal bar with icons, that can be slidable with mouse click, how can i do it?
Do i have to install something new, or can i just use bootstrap to do it? 

i already have that navbar and no slidebar. My navbar is a header component that i created, with some css to fix it, the problem is:

- if i fix the height (` overflow: auto and class="d-flex align-content-around flex-nowrap "` ) and the dowpdown menus go insise that navbar, and i have to scrool down to "select" one,
if use flex (`class=" ... flex-wrap"` )  the height is flexible the dropdowns appear outside wich is what i want, but  the nabvar is not at same height. here's the code i have.. (flexible):

```
<nav class="sidebar">
<div class=" pos-f-t fixed-header header ">
    <nav class=" navbar2 navbar-toggleable-x1  navbar-nav navbar-expand-xl">
            <div class="d-flex align-content-around flex-wrap "><!--flex-md-wrap-->
                (... menus ... )
                <li class="nav-item dropdown" ngbDropdown>
                    <a  [routerLinkActive]="['router-link-active']" role="button"  class="nav-link btn mt-1"  ngbDropdownToggle style="padding: .375rem 1rem !important; border-color: #999;"><img src="/assets/images/documents.png" alt=""></a>
                    <div class="dropdown-menu dropdown-menu-right">
                        (dropdown sub menu..)
                    </div>
                </li>
                (... menu ...)
            </div>
    </nav>
</div>
</nav>
```
css:
```
$topnav-background-color: #ffffff;
.sidebar {
    //display: grid;
    display: inline-block;
    border-radius: 0;
    position: fixed;
    z-index: 1000;
    //top: 60px;
    left: 100%;
    width: 100%;
    margin-left: -100%;
    border: none;
    border-radius: 0;
    //overflow-y: auto;
    //overflow-x: auto;
    //overflow:auto;
    background-color: transparentize($topnav-background-color,0);
    //bottom: 80%;
    -webkit-transition: all 0.3s ease-in-out;
    -moz-transition: all 0.3s ease-in-out;
    -ms-transition: all 0.3s ease-in-out;
    -o-transition: all 0.3s ease-in-out;
    transition: all 0.3s ease-in-out;
    white-space: nowrap;
    float: none;
}


.sidebar::-webkit-scrollbar {
    /* hide scroll menu*/
    display: none;

}
```

tl;dr: i want a horizontal slidable navbar with icons fixed on top of the page.

Help me please! thank you!
